### DISCUSSION TITLE
Implement KyuubiHistoryServerPlugin like HiveThriftServer2HistoryServerPlugin

### DISCUSSION THREAD

**Discussion by wForget** [apache/kyuubi#1812] [Ideas]
We can implement `KyuubiHistoryServerPlugin` like `HiveThriftServer2HistoryServerPlugin` to show kyuubi spark engine ui in SparkHistoryServer.

**Comment by yaooqinn** [comment]
looks like a good idea

**Reply by wForget** [reply]
Thanks, I will try to implement it.
